content/00.front-matter.md

```diff
@@ -0,0 +1,12 @@
+# Manubot Rootstock: Manuscript Title
+
+## Authors
+
+{% for author in authors %}
++ **{{author.full_name}}**<br>
```

Comment:
It looks like `<br>` works fine in the pdf based on your screenshot.  Given our desire to primarily support pandoc conversion to html and pdf, this should be fine.  I'm not sure whether it would create problems with docx or latex conversions.